src/components/MapSelectors/GlobalSelector.js

```diff
@@ -213,46 +213,49 @@ const GlobalSelector = ({ handleChange, extendRight }) => {
         selectedItem,
         getRootProps,
         openMenu,
-      }) => (
-        <StyledDropDownWrapper>
-          <StyledInputWrapper
-            {...getRootProps(
-              {
-                isOpen,
-              },
-              { suppressRefError: true },
-            )}
-          >
-            <StyledInputIcon>
-              <SearchIcon />
-            </StyledInputIcon>
-            <StyledInput
-              {...getInputProps({
-                isOpen,
-                onFocus: () => {
-                  openMenu();
+      }) => {
+        onIsFocusedChanged(isOpen);
```

Comment:
I considered also doing this by hoisting `Downshift` up, but that seemed like it would produce a huge diff. This solution to work fine.